storage.ml

```diff
@@ -208,6 +212,23 @@ module Make (BLOCK : Mirage_block.S) = struct
         Logs.err (fun m -> m "Error with storage: %s" err);
         Lwt.return (Error (`Msg err))
 
+  let update_user_unikernel_updates store
+      (new_update : User_model.unikernel_update) (user : User_model.user) =
+    let is_unique (u : User_model.unikernel_update) =
+      u.name <> new_update.name
```

Comment:
could we please avoid the polymorphic comparison, and be more explicit? `not (String.equal u.name new_update.name)` should be fine here.